Adyen/client.py

```diff
@@ -219,6 +245,18 @@ def call_api(self, request_data, service, action, idempotency=False,
             errorstring = """Please set your webservice password.
              You can do this by running 'Adyen.password = 'Your password'"""
             raise AdyenInvalidRequestError(errorstring)
+            # xapikey at self object has highest priority.
+            # fallback to root module
+            # and ensure that it is set.
+            if self.xapikey:
```

Comment:
the api key can become the first to check and use instead of the username password, since all our APIs are working fine with the API key only.